### PR TITLE
Closed #55 validating signup and login

### LINKED ISSUE #55: validate login and signup

### PR DESCRIPTION
**Pull request by bwuphan**

### CODE REVIEW

File: server/users/userController.js [hunk 1]
```diff
@@ -37,14 +37,14 @@ module.exports = {
...
+                req.session.user = username;
                 res.send({redirect: '/#/'});
               });
             }
           });
         //if the user already exists...
         } else {
-          console.log('Account already exists');
-          res.send({redirect: '/#/signup'});
+          res.send('Account already exists');
```
Comment (SherryH) [inline server/users/userController.js]: Will we need to do some Login html change to display this message when when the response is "Account already exists"?

Comment (bwuphan) [inline server/users/userController.js]: I think this is for the signup page isn't it?

File: client/dist/bundle.js [hunk 2]
```diff
@@ -35225,7 +35240,7 @@
 	                      null,
 	                      'Password'
 	                    ),
-	                    _react2.default.createElement('input', { type: 'password', name: 'password', className: 'form-control', placeholder: 'Password', value: this.state.password,
+	                    _react2.default.createElement('input', { type: 'password', name: 'password', className: 'form-control', placeholder: 'Please enter at least 6 characters', value: this.state.password,
```
Comment (bwuphan) [inline client/dist/bundle.js]: Great idea. I will do that. Does that basically do a git ignore or should we just add it in there

File: server/users/userController.js (hunk 1 above)
Comment (SherryH) [inline server/users/userController.js]: Yup you are right,  sign up page. We dont need to do it now though, just confirming potential future addon features!

File: client/dist/bundle.js (hunk 2 above)
Comment (SherryH) [inline client/dist/bundle.js]: just type it in terminal. It will take out the bundle.js from the committed files

**Pull request closed without merging**